src/main/java/myfirstproject/service/impl/WriteFileImpl.java

```diff
@@ -0,0 +1,21 @@
+package myfirstproject.service.impl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+import myfirstproject.model.Fruit;
+import myfirstproject.service.PreparingData;
+import myfirstproject.service.WriteFile;
+
+public class WriteFileImpl implements WriteFile {
+    @Override
+    public void writeToFile(String path, Map<Fruit, Integer> mapToWrite) {
+        PreparingData preparingData = new PrepareDataImpl();
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(path))) {
+            bufferedWriter.write(preparingData.prepare(mapToWrite));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file. " + path, e);
+        }
+    }
```

Comment:
why are we creating instance of `PreparingData` here?
can we have something like this
```suggestion
    public void writeToFile(String path, String content) {
        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(path))) {
            bufferedWriter.write(content);
        } catch (IOException e) {
            throw new RuntimeException("Can't write to file. " + path, e);
        }
    }
```